README.md

```diff
@@ -24,29 +20,12 @@ For this reason we curl the proto's and make them available in our aggregate pro
 
 ## Workflow for updating common apis
 
-### Ensure the go tool chain is setup on your host
-
-It is inordinately cumbersome to set this all up for docker. It can't be
-achieved without imposing significant inneficiencies in the workflow. For any
-engineer working on the api's it is reasonable to expect they can install the
-correct version of go for their host, and likely already have.
-
-### Use an avid/src/go.work
-
-First point avid at a clone of this repo so that it's imports will resolve
```

Comment:
Can you explain how we build against locally changed sources without committing and pushing ?

I don't know if go.work is necessar nor not post this change. But I do want to be able to build forestrie, avid and avid-events against a locally modified common-api.

With this change (and with particular reference to the deleted instructions) how do I do that ?

Reply:
I dont think anything has changed in this regard

I exercised the workflow commands in this README and all just worked.

The only change is how tools are managed not how they are used

Reply:
ok, can you please test that and show some evidence of it in the PR. but sounds good thanks